pipenv/cli/command.py

```diff
@@ -709,5 +709,24 @@ def verify(state):
     sys.exit(0)
 
 
+@cli.command(
+    short_help="Generate a requirements.txt from Pipfile.lock.",
+    context_settings=CONTEXT_SETTINGS,
+)
+@option("--dev", is_flag=True, default=False, help="Also add development requirements.")
```

Comment:
Can you please add a flag for including the hashes?
Also, some people like keeping `requirements.txt` and `requirements-dev.txt`. I think this would be nice to have ``--dev-only`` too.

Reply:
Yeah we have another ticket that was just opened about getting hashes into the requirements.txt  -- https://github.com/pypa/pipenv/issues/5025

Reply:
I will add this today